xarray/backends/rasterio_.py

```diff
@@ -282,7 +282,7 @@ def open_rasterio(filename, parse_coordinates=None, chunks=None, cache=None,
         # CRS is a dict-like object specific to rasterio
         # If CRS is not None, we convert it back to a PROJ4 string using
         # rasterio itself
-        attrs['crs'] = riods.crs.to_string()
+        attrs['crs'] = riods.crs.to_proj4()
```

Comment:
I recommend the following:

```suggestion
        attrs['crs'] = riods.crs.to_dict()
```

It's both forward and backward compatible. As far as I can tell from scanning the xarray source, the data array's crs attribute is rarely used, except as an argument to rasterio.transform where a dict should do just fine.

Reply:
@sgillies It's not just how xarray uses the crs attribute but users. The reason I made this PR was because the xarray crs attribute changed from PROJ4 to WKT.

Reply:
Just curious, why does it need to be a PROJ.4 string in xarray?

Reply:
For now, backwards compatibility. I don't think there is any other strong reason other than what the original developer chose (it is a fairly popular CRS format). If you think it should be WKT then that should probably go in its own issue.

Reply:
The main reason was serialization to NetCDF: attributes can't be dict or objects, and strings are a perfect representation of the CRS. Any other idea welcome though.